recipes/pyscenic/build.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Shellcheck

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)